src/tunnels/views.py

```diff
@@ -236,8 +236,15 @@ def get_script(
     ):
         reverse_port = server_auth_key.reverse_port
 
-        config_string = f"""$continue = $true
-echo "[+] Script started"
+        config_string = f"""echo "[+] Script start"
```

Comment:
Embed script (powershell) in a completely different language (Python) can cause chaos and hard to maintain.
I'll make a PR to implement template system for this issue, so it's OK for now.